Plots/Contours/NCL_h_lat_7.py

```diff
@@ -0,0 +1,223 @@
+"""
+NCL_h_lat_7.py
+==============
+This script illustrates the following concepts:
+   - Drawing vectors over filled contours
+   - Drawing pressure and height scales
+   - Interpolate to user specified pressure levels
+   - Using the geocat-comp method `interp_hybrid_to_pressure <https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.interp_hybrid_to_pressure.html#geocat.comp.interp_hybrid_to_pressure>`_
+   - Using a different color scheme to follow best practices for visualizations
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/h_lat_7.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/h_lat_7_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import xarray as xr
+from matplotlib import pyplot as plt
+import numpy as np
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+from geocat.comp import interp_hybrid_to_pressure
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+
+# Extract variables
+T = ds.T  # temperature (K)
+V = ds.V  # meridional wind (m/s)
+Z = ds.Z3  # geopotl. height in m
+omega = ds.OMEGA  # vert. pres. vel.(mb/day)
+lev = ds.lev  # pressure levels (millibars)
+q = ds.Q  # spec. humidity (g/kg)
```

Comment:
```suggestion
Z = ds.Z3  # geopotential height (m)
omega = ds.OMEGA  # vertical pressure velocity (mb/day)
lev = ds.lev  # pressure levels (millibars)
q = ds.Q  # specific humidity (g/kg)
```
To avoid any confusion with abbreviations, we can use the full names of the variables. The names are in the NetCDF file metadata, and you can use the `ncdump` function from the netCDF4 package to print this information in the terminal.